### ISSUE TITLE
First time install, fails

### ISSUE THREAD

**Issue by saas786** [Varying-Vagrant-Vagrants/VVV#572]
Hi,

I have installed vagrant, and cloned this repo "https://github.com/Varying-Vagrant-Vagrants/VVV/" on my local machine, and "vagrant up" it did install virtualbox image, and necessary modules php, ngnix and so on.

In the end when it was doing things for **default: Initializing grunt in WordPress develop**

it fails with this error.

```
==> default: Initializing grunt in WordPress develop... This may take a few mome
nts.
==> default: Loading "browserify.js" tasks...
==> default: ERROR
==> default: >> Error: Cannot find module 'crypto-browserify'
==> default: Loading "imagemin.js" tasks...
==> default: ERROR
==> default: >> Error: Cannot find module 'balanced-match'
==> default: Warning: Task "browserify:media" not found. Use --force to continue
.
==> default: Aborted due to warnings.
==> default: PHPMyAdmin already installed.
==> default: Cleaning the virtual machine's /etc/hosts file...
==> default: Adding domains to the virtual machine's /etc/hosts file...
==> default:  * Added vvv.dev from /srv/www/vvv-hosts
==> default:  * Added local.wordpress.dev from /srv/www/vvv-hosts
==> default:  * Added local.wordpress-trunk.dev from /srv/www/vvv-hosts
==> default:  * Added src.wordpress-develop.dev from /srv/www/vvv-hosts
==> default:  * Added build.wordpress-develop.dev from /srv/www/vvv-hosts
==> default: -----------------------------
==> default: Provisioning complete in 298 seconds
==> default: External network connection established, packages up to date.
==> default: For further setup instructions, visit http://vvv.dev
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: stdin: is not a tty
==> default: mysql stop/waiting
==> default: mysql start/running, process 22903
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: stdin: is not a tty
==> default:  * Restarting nginx nginx
==> default:    ...done.

G:\virtual box\vagrant-sites\wordpress-vvv>
```

What should I do.?

I am on windows 7, 64bit, VirtualBox 4.3.20, Vagrant 1.7.2

Thanks.

**Comment by cfoellmann** [comment]
sounds like an issue with `npm`. It sometimes has hick-ups.
Maybe try again? Remove `www/wordpress-develop` and run `vagrant provision`

**Comment by saas786** [comment]
Thanks @cfoellmann 

Tried it, but didn't worked :(.

Also I can't access **http://build.wordpress-develop.dev/**

I took a peak into the code, and found that,

```
provisions/provisions.sh
```

**line 582 - 586 has the issue**

which is

```
	if [[ ! -d /srv/www/wordpress-develop/build ]]; then
		echo "Initializing grunt in WordPress develop... This may take a few moments."
		cd /srv/www/wordpress-develop/
		grunt
	fi
```

if I comment this I see no other errors.

**Comment by cfoellmann** [comment]
Your error is showing because the grunt build process fails.
This means build.wordpress does not get build and thereby is not accessible.

I will try to test a build myself as soon as I got time

**Comment by cfoellmann** [comment]
the grunt process is not running for either

**Comment by saas786** [comment]
If I do use a stable release everything works fine. Only master / active branch has this issue.

**Comment by fa7ad** [comment]
I think grunt is responsible for my issue as well #577

**Comment by jeremyfelt** [comment]
I just went through a `vagrant provision` and then a full `vagrant up` from scratch and it appears that things are back in order. I'm wondering if it could have been related to a change in core's Gruntfile? https://core.trac.wordpress.org/browser/trunk/Gruntfile.js?rev=31373

I'll leave this open to keep an eye on it, but I think running `vagrant provision` again at this point should take care of it. If not, then remove the `wordpress-develop` directory and run `vagrant provision` again.

**Comment by fa7ad** [comment]
@jeremyfelt yes, I removed the whole thing and cloned the repo again. ran `vagrant up` again and this time there was no such message. So, consider my issue *closed*

**Comment by cfoellmann** [comment]
@jeremyfelt CLOSE

**Comment by golfhotel** [comment]
From several days I have a similar problem. Npm does not install correctly dependencies. I run three new installations of VVV but the result no changes. Wordpress-develop is not completely installed. Below is a screenshot to better understand.

![vvv-grunt](https://cloud.githubusercontent.com/assets/7843301/9891853/d4e5eb44-5c09-11e5-8461-1f6a71b01ec3.jpg)

Also verify that NPM works properly only in the ```/home/vagrant``` folder while in the ```/srv/www```  or ```/vagrant``` folder reports an error. Below the folder structure as reference; folders with asterisks are those mentioned.

```
┌─ bin
├─ boot
├─ build
├─ dev
├─ etc
├─ home
│ 	├─ ubuntu
│ 	└─ vagrant (*)
│			└─ bin
├─ inittrd.img
├─ lib
├─ lib64
├─ lost+found
├─ media
├─ mnt
├─ opt
├─ proc
├─ root
├─ run
├─ sbin
├─ srv
│ 	├─ config
│ 	├─ database
│ 	├─ log
│ 	└─ www (*)
├─ sys
├─ tmp
├─ usr
├─ vagrant (*)
│ 	├─ CHANGELOG.md
│ 	├─ config
│ 	├─ CONTRIBUTING.md
│ 	├─ database
│ 	├─ LICENSE
│ 	├─ log
│ 	├─ provision
│ 	├─ README.md
│ 	├─ Vagrantfile
│ 	├─ wp-cli.yml
│ 	└─ www
├─ var
└─ vmlinuz
```
Below the error.

![npm-grunt-error](https://cloud.githubusercontent.com/assets/7843301/9892200/e211b714-5c0c-11e5-86f4-2740082fe581.jpg)

**Comment by jeremyfelt** [comment]
I believe issues related to this should be resolved as of #969.
